src/issue.cpp

```diff
@@ -0,0 +1,338 @@
+/*
+Copyright libCellML Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "libcellml/issue.h"
+#include "libcellml/types.h"
+#include "utilities.h"
```

Comment:
Add a blank line before that `#include "utilities.h"`. I believe this is the way we (now) do it...? @hsorby?

Reply:
Done.